File: fizzbuzz.go
package main

import "fmt"

func main() {
	for i := 1; i <= 100; i++ {
		if i%3 == 0 {
			fmt.Printf("Fizz")
		}
		if i%5 == 0 {
			fmt.Printf("Buzz")
		}
		if i%3 != 0 && i%5 != 0 {
			fmt.Printf("%d", i)
		}
		fmt.Printf("\n")
	}
}

File: fizzbuzz.dart
#!/usr/bin/dart

void main() {
  for (var i = 1; i <= 100; i ++) {
    var o = "";
    if (i % 3 == 0) {
      o = "Fizz";
    }
    else if (i % 5 == 0) {
      o += "Buzz";
    }
    else {
      o = i.toString();
    }
    print(o);
  }
}